src/style.css

```diff
@@ -252,3 +252,7 @@
   font-size: 12px;
   font-weight: 500;
 }
+
+.leaflet-control-zoom.leaflet-bar {
```

Comment:
this removes a broken looking offset shadow that appears around the zoom buttons